### DISCUSSION TITLE
Looking to make a popup in my test website as a way to learn vanilla javascript

### DISCUSSION THREAD

**Discussion by DevRenamon-Alder** [community/community#139521] [Programming Help]
### Body

I'm using css grid to overlay the popup and hiding it with javascript, then revealing it when clicking on the digimon. However it wont hide. Help needed. Keep in mind I am not trying to hide it with html or css because I'm specifically attempting to put my new javascript knowledge to use. 

Here's the repository https://github.com/DevRenamon-Alder/Test-PartneredDigimon

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

edit: figured it out, it needed the defer attribute; new problem: the lower row of digimon moves up and down depending on if the popup is shown

**Comment by ebndev** [comment]
Hi @DevRenamon-Alder, thanks for being a part of the GitHub Community!

- One way you can do it is using the `style.display` property. Here's the [documentation](https://www.w3schools.com/jsref/prop_style_display.asp)

**Reply by DevRenamon-Alder** [reply]
I tried that and sadly I'm running into the same issue being that it won't hide in the first place. Thanks anyways tho.

**Reply by ebndev** [reply]
Check out the code imported into [JS Bin](https://jsbin.com/vudiwamabi/edit?html,js,output) and it was working there!

**Reply by DevRenamon-Alder** [reply]
So the way I've been checking these things is via dragging the local html file to my firefox browser. Is there like a glaring problem with that strategy that would explain why its not working on my end? I'm new to this.

**Reply by DevRenamon-Alder** [reply]
figured it out, it needed the defer attribute; new problem: the lower row of digimon moves up and down depending on if the popup is shown

**Reply by ebndev** [reply]
You could use `classList.add` instead of the `style` property then.

- Add this to your popup rule in CSS --> `display: none;` so it displays as none by default
- Create another rule with 
- `.newPopupClassName{ ` 
  - `display: block!important;` (the !important should give this more weight)
  - `position: absolute;` 
 ` }`
  - include any other styling you'll need to position the popup correctly
- In JS, when the element is clicked on --> `popup.classList.add('newPopupClassName')`
- To exit / close the popup --> `popup.classList.remove('newPopupClassName')`

**Reply by ebndev** [reply]
Documentation on the [position property](https://www.w3schools.com/css/css_positioning.asp)
